src/mongo/db/curop.cpp

```diff
@@ -641,6 +641,21 @@ void OpDebug::append(const CurOp& curop,
     b.append("rateLimit",
              executionTimeMicros >= serverGlobalParams.slowMS * 1000LL ? 1 : serverGlobalParams.rateLimit);
 
+    // We have following causes:
+    // - slow: execution time is greater than threshold value
+    // - collscan: number of docs examined is above threshold
+    // - random: randomly selected operation according to current rateLimit value
+    // - all: operaton is not slow and rateLimit does not filter anything
+    if (executionTimeMicros >= serverGlobalParams.slowMS * 1000LL) {
```

Comment:
Please take out `executionTimeMicros >= serverGlobalParams.slowMS * 1000LL` expression into separate variable as it's used above as well.